src/styles/global.css

```
@@ -3159,6 +3159,22 @@ a.tag:hover {
+.mermaid-figure {
+  margin: 0;
+}
+
+.mermaid-figure__description {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)